File: apps/plant_shop_python/models/plant.py
# models/plant.py

from dataclasses import dataclass
from datetime import datetime
from decimal import Decimal

"""
	Représente une plante dans la base de données.

	@param id Identifiant unique de la plante
	@param name Nom de la plante
	@param description Description de la plante
	@param price Prix de la plante
	@param stock Quantité en stock
	@param created_at Date de création
"""
@dataclass
class Plant:
    id: int
    name: str
    description: str
    price: Decimal
    stock: int
    created_at: datetime

File: apps/plant_shop_c-sharp/plant_shop_asp_dapper/Models/Plant.cs
namespace plant_shop_asp_dapper.Models
{
    public class Plant
    {
        public int Id { get; set; }
        public required string Name { get; set; }
        public string? Description { get; set; }
        public decimal Price { get; set; }
        public int Stock { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}
